src/feed/Feed.js

```diff
@@ -98,6 +101,12 @@ class Feed {
                 });
                 callback({ quotes: [] });
                 return;
+            } else {
+                response = await this._binaryApi.getTickHistory(dataRequest);
```

Comment:
@shayanbinary I may be wrong but i don't see `response` being used anywhere.

Reply:
@aminroosta How about the interval that bruce mention above

Reply:
@shayanbinary when the error code is `NoRealtimeQuotes` we also want to get the tick history. _Other than that_, we don't want to be retrieving the tick history; just display the error message.

Reply:
@brucebinary as `NoRealtimeQuotes` need both history and notification, I separate its condition and for the rest of errors just show a notification.